.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -20,28 +17,42 @@ The more repro details you can provide, along with a zip
 of the log files from your session, the better the chances
 are for a quick resolution.
 
+You may also want to record a GIF of the bug occurring and
+attach it here by dropping the file into the description body.
+
 -->
 
 ### System Details
 
 - Operating system name and version:
 - VS Code version:
 - PowerShell extension version:
-- Output from `$PSVersionTable`:
+- Output from `$PSVersionTable`
 
 ```
 Copy / paste the following commands into the PowerShell Integrated Console, and paste the output here:
 
-code -v
-$pseditor.EditorServicesVersion
-code --list-extensions --show-versions
-$PSVersionTable
+> code -v
```

Comment:
By adding the `>` it makes it harder to copy/paste into the PSIC.  Now you have to manually remove `>` chars in order to execute the commands.  We might also want to mention that there is a PowerShell extension command `Upload Bug Report to Github` to generate a bug report that will automatically include all of the info above.  Well, when it works ... sigh.  That's another bug to look into.

Reply:
Yeah I was vacillating between the two. I keep seeing issues where those commands aren't executed properly or the output is dumped into the issue with no formatting and is unreadable.

I was thinking maybe the `>` would prompt people to put the output underneath to break up the issue better... But maybe it's not a good idea?

There's a reference to the open bug command in troubleshooting.md, but I can put one in the template as well

Reply:
Wait the `Upload Bug Report to GitHub` is already in the template -- second line in the comment